public/app.css

```diff
@@ -37,24 +37,24 @@
   src: url('/fonts/BasierSquare-Regular.otf') format('opentype');
 }
 
-body {
+body,
+html {
   font-family: 'Calibre', sans-serif;
   font-weight: 400;
   font-size: 16px;
   margin: 0;
   padding: 0;
-  -webkit-font-smoothing: antialiased;
-  scroll-behavior: smooth;
+  /* -webkit-font-smoothing: antialiased; */

```

Comment:
Oops I think I commented this by mistake yesterday. Not sure it’s needed though?